saviSimulation/savi/jason_processing/UxVBehavior.java

```diff
@@ -252,10 +248,14 @@ protected double calculateFailureValue (double mean, double stdDev) {
 		return ((rand.nextGaussian()*stdDev)+mean);
 	}
 	
-	
-	
```

Comment:
Unclear to me why this method is called setSeed()

Reply:
@ntartania because originally it took a seed and create the random generator:
public static void setSeed(int seed) {
		if(seed != -1) {
			rand = new Random(seed);
}
}
As we need to define the random generation just once, we modified the method to just copy the value of rand defined in SaviWorldModel